docs/UserGuide.adoc

```diff
@@ -132,6 +132,22 @@ Deletes the 2nd person in the address book.
 `delete 1` +
 Deletes the 1st person in the results of the `find` command.
 
+=== Removing a tag: `remove`
```

Comment:
Could we have a more specific command name? A user might reasonably expect a `remove` command to function the same as `delete` (they mean the same thing after all). Consider `deletetag` - which also preserves consistency in the sense that all operations that delete stuff start with `delete`.